src/main/java/worldofzuul/presentation/SceneController.java

```diff
@@ -76,11 +76,22 @@ public void handleBtnHelp() throws Exception {
      * @since 1st Iteration -
      */
     public void handleBtnNextYear() throws Exception {
-        //Change window
-        GUI_Main.setRoot("next year");
-        
-        Command c = new Command(CommandWord.NEXT, "year");
-        Game.instance.nextYear(c);
+        //If 20 years have passed the game is over, and the player will get to the recap
+        if(Game.instance.getGameYear() == 20) {
```

Comment:
You should call the next year method in Game first and check its return value. If it is true show the recap page and if it is false show the next year page